featuretools/entityset/entityset.py

```diff
@@ -819,8 +819,8 @@ def normalize_dataframe(self, base_dataframe_name, new_dataframe_name, index,
 
         base_dataframe_index = index
 
-        # --> Implementation: not sure that this is the same as using vtype Categorical because we can't just set a standard tag
-        # why did we set it to variable Categorical???
+        # --> Implementation: not sure that this is the same as using ltype Categorical because we can't just set a standard tag
+        # why did we set it to Categorical???
         # and why does it get reset when we did it above??
         # transfer_types[index] = ('Categorical', set())
```

Comment:
Also added these four lines that we should probably remove - they don't help with understanding, and I don't think we've found any reason to reset the index to be categorical (commented out like 825)